src/components/SøknadsSteg/Steg/Steg.less

```diff
@@ -6,42 +6,57 @@
     
     &__knapper {
       padding: 2rem;
-      grid-area: knapper;
       display: flex;
       justify-self: center;
       flex-direction: column;
       align-items: center;
+
+      @media @mobile {
+        margin-bottom: 10rem;
+      }
+
+      @media (orientation : landscape) {
+        margin-bottom: 10rem;
+      }
   
       .avbryt {
         margin-top: 1rem;
         justify-content: center;
-        max-width: 134px;
+        width: 149px;
       }
 
       &--rad1 {
         display: flex;
         flex-direction: row;
         justify-content: center;
+        min-width: 149px;
     
         @media @mobile {
-          flex-direction: column;
-          
+          flex-direction: column-reverse;
+        }
+
+        & .neste {
+          width: 149px;
+        }
+
+        & .neste-alene {
+          width: 149px;
```

Comment:
Trenger neste og neste-alene å være to forskjellige klasser siden de har samme styling?
Ser også at width: 149px er brukt mange steder, hadde det vært mulig å trekke det ut? Burde vi kanskje bruke rem istedet for px?